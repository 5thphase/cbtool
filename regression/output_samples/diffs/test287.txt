 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_67 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_70 (xxx.xxx.xxx.xxx )...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_68 (xxx.xxx.xxx.xxx )...
