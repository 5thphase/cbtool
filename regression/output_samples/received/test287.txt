 status: Waiting for vm_66 (cloud-assigned uuid A1F4422E-9EEC-535A-A17C-B14555B506D9) to start...
 status: Waiting for vm_67 (cloud-assigned uuid 5A87C684-FD63-5024-A606-4D0D859065BF) to start...
 status: Waiting for vm_68 (cloud-assigned uuid CF0F30D4-9C5D-55D6-9CCE-28CC3F058C74) to start...
 status: Waiting for vm_69 (cloud-assigned uuid 18594A1D-D024-5515-9685-C13DDDADC7EC) to start...
 status: Waiting for vm_70 (cloud-assigned uuid 8ECF02A6-01F4-5B3A-A41C-2CF05A6B09A6) to start...
 status: Trying to establish network connectivity to vm_66 (cloud-assigned uuid A1F4422E-9EEC-535A-A17C-B14555B506D9), on IP address 146.197.226.150...
 status: Trying to establish network connectivity to vm_69 (cloud-assigned uuid 18594A1D-D024-5515-9685-C13DDDADC7EC), on IP address 222.159.67.242...
 status: Trying to establish network connectivity to vm_70 (cloud-assigned uuid 8ECF02A6-01F4-5B3A-A41C-2CF05A6B09A6), on IP address 154.103.204.13...
 status: Trying to establish network connectivity to vm_67 (cloud-assigned uuid 5A87C684-FD63-5024-A606-4D0D859065BF), on IP address 245.17.116.89...
 status: Trying to establish network connectivity to vm_68 (cloud-assigned uuid CF0F30D4-9C5D-55D6-9CCE-28CC3F058C74), on IP address 128.61.117.48...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_66 (146.197.226.150)...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_70 (154.103.204.13)...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_68 (128.61.117.48)...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_67 (245.17.116.89)...
 status: Bypassing the bootstrapping and the sending of a copy of the code tree to vm_69 (222.159.67.242)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_13...
 status: Bypassing application-specific configuration on all VMs beloging to ai_13...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_13...
AI object 557F6430-3C7B-5B07-ABAD-E18F20C9705D (named "ai_13") sucessfully attached to this experiment. It is ssh-accessible at the IP address 146.197.226.150 (A1F4422E-9EEC-535A-A17C-B14555B506D9.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |147.217.82.254  |1396711187     |none                                    |9903A609-28CB-52EA-A44C-5D1883BD08CE
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |192.7.99.62     |1396711227     |none                                    |09555B26-C0BA-54DB-BA4E-0D14B5627636
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |73.250.19.169   |1396711245     |none                                    |79085F3C-99A6-5807-9376-0F23C78DAC03
|ai_8      |ibm_daytrader  |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |53.216.249.203  |1396711333     |none                                    |F074D904-2FA1-5A07-9673-EC04DF649F30
|ai_9      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |179.153.205.166 |1396711338     |none                                    |4890FC20-91C9-57A4-B15D-4CC5E1B8EAFE
|ai_10     |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |95.58.19.90     |1396711344     |none                                    |911D8E23-ECC0-5344-8CE3-83ABEE72E125
|ai_11     |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |192.85.86.18    |1396711349     |none                                    |CC79F6CD-C5F9-537A-81AC-A1A302832C05
|ai_12     |ibm_daytrader  |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |11.112.197.219  |1396711355     |none                                    |2025BB00-3E56-599B-B562-B76A0543E4E4
|ai_13     |ibm_daytrader  |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |146.197.226.150 |1396711361     |none                                    |557F6430-3C7B-5B07-ABAD-E18F20C9705D

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |db2                 |platinum64  |58.227.217.212  |simhostc1                  |SUT                 |ai_3    |none       |5F21E651-D899-53E7-99B6-40882F0E1225
|vm_35         |was                 |iron32      |231.249.217.253 |simhostb1                  |SUT                 |ai_3    |none       |F70BCABD-CD09-57A5-AFD6-011B46782513
|vm_34         |client_daytrader    |micro32     |147.217.82.254  |simhostd1                  |LG                  |ai_3    |none       |677B851E-1A68-59FB-A2B2-A2D323126434
|vm_36         |client_daytrader    |micro32     |192.7.99.62     |simhostd1                  |LG                  |ai_4    |none       |A56EB9B7-C43A-5E94-90AD-E8F6950B2E65
|vm_37         |was                 |iron32      |161.229.45.43   |simhostc1                  |SUT                 |ai_4    |none       |3152CA65-B681-58B9-A6DF-AD94FEFF1982
|vm_38         |db2                 |platinum64  |153.128.206.131 |simhostc1                  |SUT                 |ai_4    |none       |8DBFB411-AA3B-5089-B4A8-FB3DACB8B492
|vm_39         |netclient           |platinum64  |73.250.19.169   |simhostb1                  |SUT                 |ai_5    |none       |99A818AE-CEF7-5CF3-B380-B83789A60F58
|vm_40         |netserver           |iron32      |92.138.222.180  |simhostc1                  |SUT                 |ai_5    |none       |5320C000-9824-5408-9205-436B6B3ACFA4
|vm_49         |was                 |iron32      |191.80.60.83    |simhosta1                  |SUT                 |ai_8    |none       |555D56D4-8E3B-5525-921A-745D5D7D8214
|vm_50         |lb                  |iron32      |50.225.240.14   |simhostc1                  |SUT                 |ai_8    |none       |4F2C45DE-2E63-5278-9C9F-C2E6BEBCCCFC
|vm_47         |client_daytrader    |micro32     |53.216.249.203  |simhostd1                  |LG                  |ai_8    |none       |455414AF-3EE4-5D2C-AE20-671CA9575F92
|vm_48         |was                 |iron32      |167.66.20.92    |simhostb1                  |SUT                 |ai_8    |none       |BFEB6E69-1142-5A2E-AEA6-99064BF0742F
|vm_51         |db2                 |gold32      |96.245.146.30   |simhostb1                  |SUT                 |ai_8    |none       |45DBE9B7-DC0D-5046-9466-2C233801702B
|vm_52         |db2                 |platinum64  |23.183.42.137   |simhosta1                  |SUT                 |ai_9    |none       |39468C38-84BF-5CFF-8FBF-01BEA6E66A26
|vm_54         |was                 |iron32      |56.178.116.32   |simhostc1                  |SUT                 |ai_9    |none       |3D4F6EB8-9784-573F-BA2B-567378964FD0
|vm_53         |client_daytrader    |micro32     |179.153.205.166 |simhostd1                  |LG                  |ai_9    |none       |47F07AF5-2661-548B-883B-AD48F0C2CF17
|vm_55         |db2                 |platinum64  |42.214.16.72    |simhosta1                  |SUT                 |ai_10   |none       |E7792622-AE26-53AB-888E-3E7FBDAAE2F5
|vm_57         |was                 |iron32      |30.148.61.15    |simhostb1                  |SUT                 |ai_10   |none       |F7CB537E-6EDB-510D-B376-0340F909C689
|vm_56         |client_daytrader    |micro32     |95.58.19.90     |simhostd1                  |LG                  |ai_10   |none       |5AA73B2A-93B2-5D70-9711-A30F082BB7B1
|vm_58         |was                 |iron32      |47.228.23.244   |simhostb1                  |SUT                 |ai_11   |none       |B3CED484-761D-5D01-A7D7-549426F7A747
|vm_59         |client_daytrader    |micro32     |192.85.86.18    |simhostd1                  |LG                  |ai_11   |none       |03C5AFA5-E53B-5937-808C-2FDFD9CF0832
|vm_60         |db2                 |platinum64  |168.241.184.111 |simhostb1                  |SUT                 |ai_11   |none       |363E6FCA-BEC1-5C15-98D1-BAD1E02FE28E
|vm_61         |lb                  |iron32      |53.2.126.54     |simhostc1                  |SUT                 |ai_12   |none       |08EF6265-D556-5073-9FBA-39692BDC5FAA
|vm_65         |db2                 |gold32      |62.193.44.137   |simhostb1                  |SUT                 |ai_12   |none       |77545462-84A1-5E1F-8745-533231102D64
|vm_62         |client_daytrader    |micro32     |11.112.197.219  |simhostd1                  |LG                  |ai_12   |none       |391EFC2C-F1C9-53B1-9E5A-F7F31F6EEA76
|vm_63         |was                 |iron32      |67.49.214.183   |simhostb1                  |SUT                 |ai_12   |none       |1D0F56E4-206E-5501-B2E4-7A60E304E3A5
|vm_64         |was                 |iron32      |123.95.170.203  |simhostc1                  |SUT                 |ai_12   |none       |6CDD387B-7B26-5471-98AC-45CF73F32797
|vm_66         |client_daytrader    |micro32     |146.197.226.150 |simhostd1                  |LG                  |ai_13   |none       |E92F40DC-4EE1-5A6A-8950-2EC3C3295B81
|vm_68         |was                 |iron32      |128.61.117.48   |simhostb1                  |SUT                 |ai_13   |none       |17326C84-ED3E-588C-ACB4-518705969069
|vm_67         |db2                 |gold32      |245.17.116.89   |simhostc1                  |SUT                 |ai_13   |none       |7C3AEA73-21EB-57A9-9258-3758D7B06BB4
|vm_69         |was                 |iron32      |222.159.67.242  |simhostb1                  |SUT                 |ai_13   |none       |B27B419E-6895-54D1-8B27-82A8626CD88C
|vm_70         |lb                  |iron32      |154.103.204.13  |simhostc1                  |SUT                 |ai_13   |none       |26E05751-D9F5-5A52-8BF2-F78EFD79B260

The AI object ai_13, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |true                                 
|sut                                     |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |4.11M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |507892                        
|Total Connections Received                           |10509                         
|Total Commands Processed                             |667954                        
|Number of Keys                                       |676                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |1511424                       
|Data Size                                            |291744                        
|Index Size                                           |327040                        
|Average Object Size                                  |483.01986755                  
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |32                            
|AIs                                                  |9                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |32                            
|VMs ARRIVED                                          |67                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |9                             
|AIs ARRIVED                                          |13                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |210                           

 ---------------- VMC-WIDE COUNTERS ----------------
|49282BA1-6D05-5C43-9AF0-FFE568AFD5F9 (simzone_b) VM RESERVATIONS|11                            
|F04BDEFB-2E30-560C-802C-A1FABC5D8D0C (simzone_a) VM RESERVATIONS|3                             
|9C8B8765-2143-577F-9E2E-DEDF96009D33 (simzone_d) VM RESERVATIONS|8                             
|68F26DD5-96FF-5310-9ED9-3568A17CBF4A (simzone_c) VM RESERVATIONS|10                            

