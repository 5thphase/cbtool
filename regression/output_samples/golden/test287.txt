 status: Waiting for vm_67 (cloud-assigned uuid EAD1F9A6-BB95-5B99-B73E-084C0255C8A2) to start...
 status: Waiting for vm_69 (cloud-assigned uuid 9ED5C525-4B71-5349-9D26-9B59C1F59477) to start...
 status: Waiting for vm_66 (cloud-assigned uuid BD86B24D-74EA-5871-B628-824DB37D24D2) to start...
 status: Waiting for vm_68 (cloud-assigned uuid 19532574-BFD3-5685-9DD4-A863EA83985C) to start...
 status: Waiting for vm_70 (cloud-assigned uuid 638FC8BE-B0EC-58D1-9C59-5FF4DDCD4651) to start...
 status: Trying to establish network connectivity to vm_68 (cloud-assigned uuid 19532574-BFD3-5685-9DD4-A863EA83985C), on IP address 234.228.177.203...
 status: Trying to establish network connectivity to vm_69 (cloud-assigned uuid 9ED5C525-4B71-5349-9D26-9B59C1F59477), on IP address 55.68.189.244...
 status: Trying to establish network connectivity to vm_70 (cloud-assigned uuid 638FC8BE-B0EC-58D1-9C59-5FF4DDCD4651), on IP address 25.107.250.88...
 status: Trying to establish network connectivity to vm_66 (cloud-assigned uuid BD86B24D-74EA-5871-B628-824DB37D24D2), on IP address 249.113.213.158...
 status: Trying to establish network connectivity to vm_67 (cloud-assigned uuid EAD1F9A6-BB95-5B99-B73E-084C0255C8A2), on IP address 233.222.85.98...
 status: Bypassing the sending of a copy of the code tree to vm_68 (234.228.177.203)...
 status: Bypassing the sending of a copy of the code tree to vm_67 (233.222.85.98)...
 status: Bypassing the sending of a copy of the code tree to vm_70 (25.107.250.88)...
 status: Bypassing the sending of a copy of the code tree to vm_66 (249.113.213.158)...
 status: Bypassing the sending of a copy of the code tree to vm_69 (55.68.189.244)...
 status: Bypassing generic VM post_boot configuration on all VMs belonging to ai_13...
 status: Bypassing application-specific configuration on all VMs beloging to ai_13...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_13...
AI object BDE28905-AA8C-5E61-8999-63B82C14FCB1 (named "ai_13") sucessfully attached to this experiment. It is ssh-accessible at the IP address 249.113.213.158 (BD86B24D-74EA-5871-B628-824DB37D24D2.simcloud.com).
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |14.212.26.119   |1395352750     |none                                    |41663C6B-21F9-56C0-8839-7F708E06CC24
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |105.193.238.135 |1395352792     |none                                    |73668951-3F95-5B73-9D70-0FE65E27787B
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |46.120.12.78    |1395352810     |none                                    |0AC8E248-AA9D-5133-B466-18B2D3E818AD
|ai_8      |ibm_daytrader  |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |23.30.205.12    |1395352897     |none                                    |DE09BBD1-9C5C-580E-A91E-E1DCE80D3CEF
|ai_9      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |18.68.195.97    |1395352903     |none                                    |5E6EC969-A650-5479-99FC-E03F2DDE678F
|ai_10     |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |54.191.213.250  |1395352909     |none                                    |450FB108-83BE-5EA5-AC93-FBAA57C81696
|ai_11     |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |5.145.59.26     |1395352915     |none                                    |601116EF-7B45-5810-B62C-B5D06E574605
|ai_12     |ibm_daytrader  |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |251.220.22.193  |1395352922     |none                                    |5BE1BD2F-3B1F-5F89-AB00-969A9FC2D24A
|ai_13     |ibm_daytrader  |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |249.113.213.158 |1395352929     |none                                    |BDE28905-AA8C-5E61-8999-63B82C14FCB1

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |db2                 |platinum64  |142.181.106.103 |simhosta1                  |SUT                 |ai_3    |none       |F876CACE-B5E3-5308-B24C-9159D9ED4472
|vm_35         |was                 |iron32      |113.234.36.124  |simhostb1                  |SUT                 |ai_3    |none       |BF6F4EBC-C9D7-5E3E-B575-ED8EDD0C15DB
|vm_34         |client_daytrader    |micro32     |14.212.26.119   |simhostd1                  |LG                  |ai_3    |none       |2E522884-17F1-522A-BDD0-09A2D4AA77AB
|vm_36         |client_daytrader    |micro32     |105.193.238.135 |simhostd1                  |LG                  |ai_4    |none       |3091A18D-1AFD-548F-86A4-19E1AFED6156
|vm_37         |was                 |iron32      |40.4.181.6      |simhostb1                  |SUT                 |ai_4    |none       |CEDF3CF0-449C-565C-93AD-E1A6A0F08E02
|vm_38         |db2                 |platinum64  |67.136.45.109   |simhosta1                  |SUT                 |ai_4    |none       |A1A2CB87-DE3B-552C-B5A2-235FEED36276
|vm_39         |netclient           |platinum64  |46.120.12.78    |simhostc1                  |SUT                 |ai_5    |none       |1C7167A9-B3FE-50FE-B1CC-3BF761DFDAF3
|vm_40         |netserver           |iron32      |36.111.14.206   |simhostc1                  |SUT                 |ai_5    |none       |435B009E-54BF-578D-8755-E0F8471826C9
|vm_47         |lb                  |iron32      |80.21.168.8     |simhostc1                  |SUT                 |ai_8    |none       |782FEB94-27F7-593F-9FB3-A52FF38F18A5
|vm_49         |client_daytrader    |micro32     |23.30.205.12    |simhostd1                  |LG                  |ai_8    |none       |46938D3C-81A0-5D63-8A81-7FB4923AA279
|vm_48         |was                 |iron32      |59.126.203.46   |simhostc1                  |SUT                 |ai_8    |none       |55AE598F-7328-5370-9994-04A268FED528
|vm_50         |was                 |iron32      |106.64.30.81    |simhosta1                  |SUT                 |ai_8    |none       |CEF215C1-5184-52E7-8FF3-6DB8BD8D9705
|vm_51         |db2                 |gold32      |215.243.38.206  |simhostb1                  |SUT                 |ai_8    |none       |95176214-2E38-5FA2-A5B2-F8AB46CBEEC7
|vm_54         |client_daytrader    |micro32     |18.68.195.97    |simhostd1                  |LG                  |ai_9    |none       |36FD0D15-4376-5EFC-885C-187EE0ED24D4
|vm_52         |was                 |iron32      |50.159.181.245  |simhostc1                  |SUT                 |ai_9    |none       |77A7B5E9-FA03-5868-A094-7557559BEF40
|vm_53         |db2                 |platinum64  |6.163.120.119   |simhostc1                  |SUT                 |ai_9    |none       |26EBC941-0AE8-5059-8A95-B04B06BA4357
|vm_55         |client_daytrader    |micro32     |54.191.213.250  |simhostd1                  |LG                  |ai_10   |none       |D060D852-C94C-5E75-8F10-DA4FB929B6F3
|vm_56         |was                 |iron32      |66.238.84.163   |simhostc1                  |SUT                 |ai_10   |none       |782F34D0-D825-589B-8AA0-B4EA584E1640
|vm_57         |db2                 |platinum64  |72.111.240.228  |simhosta1                  |SUT                 |ai_10   |none       |C7E3CC2F-C1FB-5508-92C7-20EB5AFB7F32
|vm_58         |client_daytrader    |micro32     |5.145.59.26     |simhostd1                  |LG                  |ai_11   |none       |EC863077-D5DC-5CBE-B6C2-EF9F51C2759A
|vm_59         |was                 |iron32      |252.7.136.6     |simhostc1                  |SUT                 |ai_11   |none       |93F361E0-BF5D-5FA6-B610-75C56D09AE4C
|vm_60         |db2                 |platinum64  |132.240.230.189 |simhosta1                  |SUT                 |ai_11   |none       |9A8547A1-A950-50B9-B4FE-593D5EA19BA5
|vm_61         |client_daytrader    |micro32     |251.220.22.193  |simhostd1                  |LG                  |ai_12   |none       |AD5639D9-A02B-53AE-B0E3-0CB4ED978425
|vm_62         |lb                  |iron32      |29.234.12.79    |simhostb1                  |SUT                 |ai_12   |none       |533FB00B-AEBD-5D1D-BEC4-AF2C2CBE0CDD
|vm_63         |was                 |iron32      |28.102.75.208   |simhostc1                  |SUT                 |ai_12   |none       |A55AD788-137E-57D0-9630-95D091D24224
|vm_65         |db2                 |gold32      |15.242.158.185  |simhostb1                  |SUT                 |ai_12   |none       |01C01495-8E13-5AE4-A188-A7F12AD23C2B
|vm_64         |was                 |iron32      |186.227.245.21  |simhosta1                  |SUT                 |ai_12   |none       |5572BF5C-44E5-58CB-9EFA-936B04245139
|vm_66         |client_daytrader    |micro32     |249.113.213.158 |simhostd1                  |LG                  |ai_13   |none       |A6650270-B6B4-5C64-8A2C-B4FBA6204A64
|vm_69         |was                 |iron32      |55.68.189.244   |simhostc1                  |SUT                 |ai_13   |none       |9766C7B7-4DA0-5EAD-9972-9FCE1CEBD483
|vm_68         |lb                  |iron32      |234.228.177.203 |simhosta1                  |SUT                 |ai_13   |none       |DF13A7CB-B86B-5988-861C-8844B060BC60
|vm_70         |db2                 |gold32      |25.107.250.88   |simhostc1                  |SUT                 |ai_13   |none       |9AFA1DE6-840C-5E83-A0C1-D9B9E858A4FA
|vm_67         |was                 |iron32      |233.222.85.98   |simhostb1                  |SUT                 |ai_13   |none       |1D39B61F-1EA1-5AF6-BCFA-6387B779FA39

The AI object ai_13, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |true                                 
|sut                                     |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |4.10M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |29644                         
|Total Connections Received                           |1282                          
|Total Commands Processed                             |1597464                       
|Number of Keys                                       |676                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |1511424                       
|Data Size                                            |292448                        
|Index Size                                           |327040                        
|Average Object Size                                  |483.385123967                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |32                            
|AIs                                                  |9                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |32                            
|VMs ARRIVED                                          |67                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |9                             
|AIs ARRIVED                                          |13                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |210                           

 ---------------- VMC-WIDE COUNTERS ----------------
|46D4622A-F4FA-5954-A362-7BB01336493E (simzone_b) VM RESERVATIONS|6                             
|A01EE257-BEE4-5C25-A4B0-57EA67D25FB3 (simzone_c) VM RESERVATIONS|11                            
|98E867EB-38E0-596F-B000-DDE51A5C4BA9 (simzone_d) VM RESERVATIONS|8                             
|D6527383-EEEF-5F20-AFDD-FD9F013A7407 (simzone_a) VM RESERVATIONS|7                             

